internal/backend/auth/authjwttokens/jwt.go

```diff
@@ -0,0 +1,91 @@
+package authjwttokens
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	j "github.com/golang-jwt/jwt/v5"
+	"github.com/google/uuid"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/auth/authtokens"
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+const issuer = "ak"
+
+var (
+	method   = j.SigningMethodHS256
+	hashSize = method.Hash.Size()
+)
+
+type Config struct {
+	SignKey string `koanf:"sign_key"`
+}
+
+type tokens struct {
+	signKey []byte
+}
+
+var Configs = configset.Set[Config]{
+	Dev: &Config{
+		SignKey: strings.Repeat("00", hashSize),
+	},
+}
+
+func New(cfg *Config) (authtokens.Tokens, error) {
+	key, err := hex.DecodeString(cfg.SignKey)
+	if err != nil {
+		return nil, fmt.Errorf("invalid sign key: %w", err)
+	}
+
+	if len(key) != hashSize {
+		return nil, fmt.Errorf("invalid key len: %d != desired %d", len(key), hashSize)
+	}
+
+	return &tokens{signKey: key}, nil
+}
+
+func (js *tokens) Create(user sdktypes.User) (string, error) {
+	uj, err := user.MarshalJSON()
+	if err != nil {
+		return "", fmt.Errorf("marshal: %w", err)
+	}
+
+	uuid, err := uuid.NewV7()
```

Comment:
what's the purpose of a new uuid each time user creates a token ?
id should be related to the user somehow, and deterministic

Reply:
1. consecutive tokens generated in the same second will differ.
2. ability to cancel it in the future, per specific token.

Reply:
not sure I get it

this is the id for the token ? 
you can just use the token it self as it is unique

if it's something that should reference the user - it should be deterministic by user characteristics